### DISCUSSION TITLE
Please update Phaser version to 3.60

### DISCUSSION THREAD

**Discussion by JCtapuk** [Annoraaq/grid-engine#388] [General]
The current version is 3.60 but your package does not allow you to install as you specified ~ 3.55.2

**Comment by Annoraaq** [comment]
@JCtapuk I am already working on it. There are some compatibility issues that need to be solved.

**Reply by JCtapuk** [reply]
> @JCtapuk I am already working on it. There are some compatibility issues that need to be solved.

I forcibly installed via --force and so far it works fine, I haven't used it all yet, but just spawned objects in the editor

**Reply by Annoraaq** [reply]
I have just released a new version that includes compatibility with Phaser 3.60. It was a large change (not because of the Phaser update but because of the pathfinding optimizations in this version), so I hope there are no bugs introduced.

**Reply by JCtapuk** [reply]
> I have just released a new version that includes compatibility with Phaser 3.60. It was a large change (not because of the Phaser update but because of the pathfinding optimizations in this version), so I hope there are no bugs introduced.

good.
